modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -8,7 +8,7 @@ Dex is installed by default for all the Argo CD instances created by the Operato
 
 To configure Keycloak, follow these steps:
 
-. Delete the Dex configuration by removing the following section from the Argo CD Custom Resource (CR), and save the CR: 
+. Delete the Dex configuration by removing the `.spec.sso.dex` field from the Argo CD Custom Resource (CR), and save the CR: 
```

Comment:
```suggestion
. Delete the Dex configuration by removing the `.spec.sso.dex` parameter from the Argo CD custom resource (CR), and save the CR: 
```

Reply:
custom resource does not follow any capitalization. So, we can keep it generic (in small letters) and still use the acronym.